invenio_administration/assets/semantic-ui/js/invenio_administration/src/components/Formatter.js

```diff
@@ -39,6 +43,7 @@ Formatter.propTypes = {
   resourceSchema: PropTypes.object.isRequired,
   result: PropTypes.object.isRequired,
   property: PropTypes.string.isRequired,
+  truncate: PropTypes.number,
```

Comment:
Minor: Maybe set a defaultProp for the truncate so that github doesn't complain :wink:

Reply:
Didn't see it thanks.